Sources/Specs/guessManagerSpec.swift

```diff
@@ -0,0 +1,57 @@
+import SnowManLogic
+import Swiftest
+
+class GuessManagerSpec: Swiftest.Spec {
+  	let spec = describe("guess logic") {
+        	let guessManager = GuessManager(totalIncorrectGuessesAllowed: 10)
```

Comment:
constants are subject to polluted state that can impacts other tests, you should make this (and other similar instances throught the tests) a variable, and create it in a beforeEach block rather